OpenLIFUData/OpenLIFUData.py

```diff
@@ -736,6 +813,30 @@ def clear_plan(self,  clean_up_scene:bool = True) -> None:
         if clean_up_scene:
             plan.clear_nodes()
         self.getParameterNode().loaded_plan = None
+    
+    @display_errors
+    def add_subject_to_database(self, subject_name, subject_id): 
+        """ Adds new subject to loaded openlifu database.
+
+        Args:
+            subject_name: name of subject to be added (str)
+            subject_id: id of subject to be added (str)
+        """
+
+        newOpenLIFUSubject = openlifu_lz().db.subject.Subject()
+        newOpenLIFUSubject.name = subject_name
+        newOpenLIFUSubject.id = subject_id
```

Comment:
not too important but you should be able to construct it in one go:
```suggestion
        newOpenLIFUSubject = openlifu_lz().db.subject.Subject(
            name = subject_name,
            id = subject_id,
        )
```